src/components/ImageUpload/ImageUpload.tsx

```diff
@@ -65,15 +66,15 @@ export function ImageUpload({
   );
 
   const { uploadToCF, files: imageFiles } = useCFImageUpload();
-  const [files, filesHandlers] = useListState<CustomFile>(value);
+  const [files, filesHandlers] = useListState<CustomFile>(Array.isArray(value) ? value : []);
```

Comment:
Should we wrap `value` in a array in case it isn't an array in this case?
```suggestion
  const [files, filesHandlers] = useListState<CustomFile>(Array.isArray(value) ? value : [value]);
```

Reply:
In this case, no. If our value isn't an array it will be an empty string or null. We don't want those...